pkg/cmd/admin/policy/policy.go

```diff
@@ -115,7 +115,7 @@ func getUniqueName(basename string, existingNames *sets.String) string {
 
 // RoleBindingAccessor is used by role modification commands to access and modify roles
 type RoleBindingAccessor interface {
-	GetExistingRoleBindingsForRole(roleNamespace, role string) ([]*authorizationapi.RoleBinding, error)
+	GetExistingRoleBindingsForRole(roleNamespace, role, rb string) ([]*authorizationapi.RoleBinding, error)
```

Comment:
The new method would replace both uses of GetExistingRoleBindingsForRole(), so maybe it should be renamed. Eg., GetRoleBindingByRoleNames().